src/api_recievers/influxqueries.js

```diff
@@ -0,0 +1,204 @@
+// const { time } = require("console");
+// const { start } = require("repl");
+
+const { currentTime, InfluxDB, Point } = require("@influxdata/influxdb-client");
+
+const log = reuqire('../util/logger');
+
+
+const token = process.env.INFLUXDB_TOKEN;
+const url = 'http://localhost:8086';
```

Comment:
We should probably make the port we host the DB on an environment variable, called DB_PORT or something